ia/src/ml_agent.py

```
@@ -249,13 +262,13 @@ def strategy(client):
         protocole.execute_command(client, utils.INVENTORY, None)
         return
-    
+
     if food_count < utils.CRITICAL_FOOD:
-       client.status = "critique"
+        client.status = "critique"
 
     if client.status == "critique":
-       if food_count > utils.GOOD_FOOD:
-          client.status = "good"
+        if food_count > utils.GOOD_FOOD:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)